<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftPanel</class>
 <widget class="QFrame" name="LeftPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>42</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#LeftPanel {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(179, 210, 230, 255), stop:1 rgba(240, 240, 240, 255));
border-color: rgb(188, 188, 188);
border: 0px solid;
margin: 0; 
padding: 0
;}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <property name="lineWidth">
   <number>1</number>
  </property>
  <property name="midLineWidth">
   <number>0</number>
  </property>
  <widget class="QToolButton" name="fileButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>105</y>
     <width>38</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>file manger</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/filen.png</normaloff>:/images/filen.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>55</y>
     <width>38</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>tool</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/tools.png</normaloff>:/images/tools.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="cloudsButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>155</y>
     <width>38</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>cloud disk</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/cloud.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="expButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>5</y>
     <width>38</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>show page panel</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="shareButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>432</y>
     <width>38</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>share your document</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/share.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>332</y>
     <width>38</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>system settings</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/setting.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="optButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>382</y>
     <width>38</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>show attribution panel </string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/attrib.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="userButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>282</y>
     <width>38</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>user center</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/user.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
